notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -194,7 +194,7 @@
         "    \n",
```

Comment:
<h3>Create Pipeline for evaluations -&gt;</h3><h3>Create pipeline for evaluations</h3><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>

Reply:
<p>done</p>